src/main/frontend/pipeline-console-view/pipeline-console/main/pipeline-console.scss

```diff
@@ -1,5 +1,10 @@
 :root {
   --card-background: hsl(212, 30%, 96%);
+  --step-bg-success: var(--alert-success-bg-color);
+  --step-bg-unstable: var(--alert-warning-bg-color);
+  --step-bg-failure: var(--alert-danger-bg-color);
+  --step-bg-aborted: var(--medium-grey);
+  --step-bg-paused: var(--alert-info-bg-color);
```

Comment:
can we use the semantic colours instead?
https://weekly.ci.jenkins.io/design-library/Colors/

could submit a PR to add a few more to Jenkins core for the other states if needed or use the others temporarily

Reply:
> on the `buildPlugin` issue, do you think it's regressed any here or should be fixed separately?

Hard to say. The backend behaviour is the same (why is what's causing the steps to be listed under the wrong parent). The front-end changes make it a little harder to find the steps as before they were listed as siblings in the tree, but now they are only visible as children of the grandparent).

Reply:
On second thought, as a separate PR, so they can be reverted independently if needed.